tokenlist.json

```diff
@@ -323,96 +323,59 @@
-      "address": "0x341b332b35441c37c1211f02F2C03c1516ac3AC8",
+      "address": "0xEA6B04272f9f62F997F666F07D3a974134f7FFb9",
       "decimals": 6,
       "name": "USDC",
       "symbol": "USDC",
       "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/usd-coin-usdc-logo.svg"
     },
     {
       "chainId": 245022934,
-      "address_spl": "2FPyTwcZLUg1MDrwsyoP4D6s1tM7hAkHYRjkNb5w6Pxk",
-      "address": "0xbd5C9b22c9e0C41fE93aA863Da1402306A76d4fe",
-      "decimals": 6,
-      "name": "Wrapped Ethereum (Sollet)",
-      "symbol": "ETH",
-      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/ethereum-eth-logo.svg"
+      "address_spl": "So11111111111111111111111111111111111111112",
+      "address": "0x5f38248f339Bf4e84A2caf4e4c0552862dC9F82a",
+      "decimals": 9,
+      "name": "Wrapped SOL",
+      "symbol": "wSOL",
+      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/solana-wsol-logo.svg"
     },
     {
       "chainId": 245022934,
-      "address_spl": "qfnqNqs3nCAHjnyCgLRDbBtq4p2MtHZxw8YjSyYhPoL",
-      "address": "0x7Db49A319636818f38dc2d12B51b13192e1b9141",
+      "address_spl": "7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs",
+      "address": "0xcFFd84d468220c11be64dc9dF64eaFE02AF60e8A",
       "decimals": 8,
-      "name": "Wrapped BTC (Wormhole)",
-      "symbol": "wWBTC",
-      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/wrapped-bitcoin-wbtc.svg"
-    },
-    {
-      "chainId": 245022934,
-      "address_spl": "dK83wTVypEpa1pqiBbHY3MNuUnT3ADUZM4wk9VZXZEc",
-      "address": "0x142Ec1Db2dc73932AE590C429D2BFd13732b70e2",
-      "decimals": 6,
-      "name": "Wrapped Aave",
-      "symbol": "AAVE",
-      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/aave-aave-logo.svg"
-    },
-    {
-      "chainId": 245022934,
-      "address_spl": "ApmXkxXCASdxRf3Ln6Ni7oAZ7E6CX1CcJAD8A5qBdhSm",
-      "address": "0x633AC0dE8749f6c2fACb73665a05c622a272f291",
-      "decimals": 9,
-      "name": "Curve DAO Token (Wormhole)",
-      "symbol": "wCRV",
-      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/curve-dao-logo.svg"
+      "name": "Wrapped Ether (Wormhole)",
+      "symbol": "WETH",
+      "logoURI": ""
     },
     {
       "chainId": 245022934,
-      "address_spl": "AR1Mtgh7zAtxuxGd2XPovXPVjcSdY3i4rQYisNadjfKy",
-      "address": "0xa2b3D9E1E92Af208Ed6bc41d4428e500569Af66F",
+      "address_spl": "9n4nbM75f5Ui33ZbPYXn59EwSgE8CGsHtAeTH5YFeJ9E",
+      "address": "0x54EcEC9D995A6CbFF3838F6a8F38099E518805d7",
       "decimals": 6,
-      "name": "Wrapped SUSHI (Sollet)",
-      "symbol": "SUSHI",
-      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/sushiswap-sushi-logo.svg"
+      "name": "Wrapped Bitcoin (Sollet)",
+      "symbol": "BTC",
+      "logoURI": ""
```

Comment:
@oable please, provide logoURL

Reply:
I added icon. Will available by link after merging to main branch.